### ISSUE TITLE
User menu can go off screen

### ISSUE THREAD

**Issue by bendk** [pculture/unisubs#3421]
Steps to reproduce:
  - Size your browser at 1440px width
  - Open the user menu
  - There's not enough width to display it, which causes a really weird effect.

**Comment by bendk** [comment]
Fixed in the gh-3421 branch.

**Comment by PCF-Testing** [comment]
I have trouble reproducing this on production. Here is what I see with the browser set to 1440 px width. Is the issue specific to particular OS/browser?

![screenshot from 2018-09-20 07 47 48](https://user-images.githubusercontent.com/288455/45826662-bceddc00-bca9-11e8-8e0f-8b7b2c1aba65.png)

**Comment by bendk** [comment]
Yes at 1440px, the content should go almost up to the edge of the width.  The general steps to reproduce are:

 - Adjust the width until you get to the biggest size for the page content
 - Stop just after you the browser is wide enough.  For me this was 1440, but maybe it's a bit different depending on the browser.
 - Try the user menu.

**Comment by PCF-Testing** [comment]
Okay, although I did not see much weirdness on production at screen width near 1440px, the user menu on the branch seems to be wider at most positions, so this should count as an improvement. Moving to Needs PR queue.

**Comment by bendk** [comment]
This one is simple enough that I just merged it directly to dev.
